source/platforms/kubeflow.md

```diff
@@ -1,2 +1,233 @@
 KubeFlow
 ========
+
+You can use RAPIDS with KubeFlow in a single pod with [KubeFlow Notebooks](https://www.kubeflow.org/docs/components/notebooks/) or you can scale out to many pods on many nodes of the Kubernetes cluster with the [dask-operator](/tools/kubernetes/dask-operator).
+
+## KubeFlow Notebooks
+
+The [RAPIDS docker images](/tools/rapids-docker) can be used directly in KubeFlow Notebooks with no additional configuration. To find the latest image head to [the RAPIDS install page](https://rapids.ai/start.html#get-rapids), as shown in below, and choose a version of RAPIDS to use. Typically we want to choose the container image for the latest release.
+
+```{figure} /images/kubeflow-rapids-install.png
+---
+alt: Screenshot of the RAPIDS release sector with the Docker image selected
+---
+
+Verify the Docker image is selected when installing the latest RAPIDS release
+```
+
+Be sure to match the CUDA version in the container image with that installed on your Kubernetes nodes. The default CUDA version installed on GKE Stable is 11.4 for example, so we would want to choose that. From 11.5 onwards it doesn’t matter as they will be backward compatible. Copy the container image name from the install command (i.e. `rapidsai/rapidsai-core:22.06-cuda11.5-runtime-ubuntu20.04-py3.9`).
+
+Now in KubeFlow, access the Notebooks tab on the left and click “New Notebook”.
```

Comment:
Might be good to note what version of KubeFlow these instructions & images were generated in